README.md

```diff
@@ -11,31 +11,40 @@ want to print the results and that purpose anonymous object serves just fine.
-earned the most/least money and what city sold the most/least items?  
-
---- Extra challenge:  
-3) Daily money earned for specific shop.  
-4) What items were sold in what shop, at what price and when (file, named after shop).
-Can get a specific shop as well.  
-
-The following commands will invoke the following cases:  
-1) time (returns stats for full day); time 00:00-23:59 (returns stats between 00:00 and 23:59)  
-2) city -money -max (city that earned most money); city -money -min (city that earned the least money); 
-city -items-max (city that sold the most items); city -items -min (city that sold the least items)  
-3) daily "KwikiMart" (gets daily money earned by "KwikiMart")   
-4) full (get what did each shop sell); full "KwikiMart" (gets all items sold by "KwikiMart"). 
-Prints all items of one shop first, then next shop items.  
-
-Every command should start with a file which contains input and end with a file that contains output.  
-For example: "input/Transactions.csv" time "time.csv"
-Results should be printed to a file in .csv format.  
-
-2), 3) and 4) supports sorting by shop name (or city name) in either ascending or descending order. If not sorting is provided, it should sort in ascending order. Sorting is just 1 extra arg: -asc (sorts in ascending order by shop name); -desc (sorts in descending order by shop name)
+1) By time (`time`)
+    - how many items have been bought during every hour of time of day,
+    - how much money did every hour total (on average), 
+    - and get rush hour (most mony earned on average).
+
+Support getting items sold count and money earned for a selected range of hours as well.
+
+2) What city (can be parsed from address) earned the most/least money and what city sold the most/least items? (`city [-min/-max] [-items/-money]`)
+
+--- Extra challenge:
+
+3) Daily money earned for specific shop. (`daily Shop Name`)
+
+4) What items were sold in what shop, at what price and when (file, named after shop).  (`full`)
+
+Results should be printed to a file in .csv format.
+
+(3) and (4) supports sorting by shop name (or city name) in either ascending or descending order. If not sorting is provided, it should sort in ascending order. Sorting is just 1 extra arg: -asc (sorts in ascending order by shop name); -desc (sorts in descending order by shop name)
+
+### Example Command Invocations
+
+Here's a list of example command invocations to get you started. It is not an exhaustive list of required supported commands.
+
+1) `"path/to/input" "time" "path/to/output"` - Output the report specified in requirement 1.
+
+2) `"path/to/input" "time 11:00-17:00" "path/to/output"` - Output the report specified in requirement 1, filtered on transactions between the time of day 11:00 - 17:00.
+
+3) `"path/to/input" "city -money -max" "path/to/output"` - Output the report specified in requirement 2, determining the city where the most money is earned. 
```

Comment:
Example invocations. Because clarity is nice. :)

Reply:
100% yes!